lib/models/finamp_models.dart

```diff
@@ -2191,11 +2191,11 @@ enum LyricsFontSize {
 @HiveType(typeId: 72)
 enum KeepScreenOnOption {
   @HiveField(0)
-  disabled,
+  whileLyrics,
   @HiveField(1)
   whilePlaying,
   @HiveField(2)
-  whileLyrics;
+  disabled;
```

Comment:
I think this doesn't actually change the default, just in which order the options appear. You can keep the order how it was, starting with `disabled`.  
To change the default, change it here: https://github.com/jmshrv/finamp/pull/858/files#diff-00fd4ef72cab57c7db5c49ff8cd257918492ca5650d72bec14b3ee8560fcad54R114

Reply:
yea, I looked up how to set default and I guess it's something like calling a constructor with the value?

Reply:
There's no way to set the default of an enum. But we use the enum as a value for a setting, and there's a place in `finamp_models.dart` where we specify the default settings value to be used, if no value has been selected yet.  
This will not change the selected value after the first start, unless you clear the app data and thereby delete the database.